source/logic/bounties.js

```diff
@@ -1,8 +1,9 @@
 const { Sequelize, Op } = require("sequelize");
-const { Guild, GuildMember } = require("discord.js");
+const { Guild, GuildMember, time } = require("discord.js");
```

Comment:
```suggestion
const { Guild, GuildMember } = require("discord.js");
```
`time` looks like an accidental import here, but it's a cool formatter we might want to keep in mind in the future.
![image](https://github.com/user-attachments/assets/80adbe0b-f538-4339-8d61-aa59d4f1aef2)

Reply:
This almost certainly happened with an unfortunate autocomplete with the new time calculation code.